tests/core/pyspec/eth2spec/test/helpers/random.py

```diff
@@ -7,6 +7,7 @@
 
 
 def set_some_new_deposits(spec, state, rng):
+    queuing_indices = []
```

Comment:
the use of the word `queuing` here is a bit confusing imo bc these indices are *not* in the activation queue, they have just deposited... perhaps call this list `deposited_indices` instead?

i would consider the indices in line 21 to be the "queuing indices"